markdown/org/docs/faq/tapering-seams/en.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Why do my seam allowances taper on my patterns?
+---
+
+Freesewing generates the seam allowances by offsetting from the seam paths. This means that when there is a corner in your pattern and the two offset seam allowances converge, it would require a lot more code to make the offset seam allowance come to a point. Instead, the offset seam allowances come to the end of the seam paths and then are connected in a straight line, making a trimmed or tapered corner. 
+
+If you want you could simply cut the seam allowance corners properly when cutting your printed pattern or when cutting your fabric. You do not have to cut along the tapered lines. 
```

Comment:
nit: suggest not using the work 'properly'. Some might consider it to be an improvement for ease of sewing 🤷‍♀️

I'm also wondering if anyone has actually incorporated this 'feature' into their sewing instructions? Could it effect how the pieces line up? Just curious.

Reply:
One of examples where you can see this is Holmes. The crown doesn't have the tapered seam because it would affect the strength of said seam.